app/src/main/res/values/styles.xml

```diff
@@ -429,12 +426,9 @@
   </style>
   <!-- Continue Lesson Button -->
   <style name="ContinueLessonButton" parent="TextAppearance.AppCompat.Widget.Button">
-    <item name="android:layout_width">144dp</item>
+    <item name="android:layout_width">0dp</item>
     <item name="android:layout_height">wrap_content</item>
     <item name="android:drawablePadding">4dp</item>
-    <item name="android:paddingEnd">12dp</item>
-    <item name="android:paddingStart">12dp</item>
-    <item name="android:minWidth">144dp</item>
```

Comment:
I think the minWidth attribute here and above are important to keep as they prevent the buttons from being smaller than the specified size, unless you can provide a justification for removing them.